bazel/repositories.bzl

```diff
@@ -0,0 +1,12 @@
+load("@com_github_ray_project_ray//java:dependencies.bzl", "gen_java_deps")
+load("@com_github_nelhage_rules_boost//:boost/boost.bzl", "boost_deps")
+load("@com_github_jupp0r_prometheus_cpp//:repositories.bzl", "prometheus_cpp_repositories")
+load("@com_github_ray_project_ray//bazel:python_configure.bzl", "python_configure")
+load("@com_github_checkstyle_java//:repo.bzl", "checkstyle_deps")
+
+def ray_repositories():
```

Comment:
`ray_repositories` looks not that meaningful. How about renaming to `ray_deps_build_all` (also rename the file name) and move `ray_deps_setup()` to this function instead of calling it in `WORKSPACE`? In this case, all dependency is build in one function.

Reply:
As discussed with @ruifangChen offline, it is hard to put `ray_deps_setup` in the function of `ray_deps_build_all` and `ray_deps_setup` should be put ahead of `ray_deps_build_all` to setup the dependencies. Therefore, she will not merge this function, which is fine to me.